README.md

```diff
@@ -130,17 +129,18 @@ Please read our [Contributing Guide](./CONTRIBUTING.md) before submitting a Pull
 
 ## Support
 
-For more information on the upcoming version, please take a look to our [ROADMAP](https://github.com/strapi/strapi/projects).
+For more information on the upcoming version, please take a look at our [ROADMAP](https://github.com/strapi/strapi/projects).
```

Comment:
Should move the Roadmap link to product board

Reply:
This sentence is actually totally unnecessary.  Later we say practically the same thing but better. So I deleted this instance.